tests/e2e/testsLibrary/LanguageServerTests.ts

```diff
@@ -75,21 +75,42 @@ export class LanguageServerTests {
         });
     }
 
-    public codeNavigation(openedFile: string, line: number, char: number, codeNavigationClassName: string, timeout : number = TimeoutConstants.TS_EDITOR_TAB_INTERACTION_TIMEOUT) {
-        test('Codenavigation', async () => {
+    public goToDefinition(openedFile: string, line: number, char: number, codeNavigationClassName: string, timeout : number = TimeoutConstants.TS_EDITOR_TAB_INTERACTION_TIMEOUT) {
+        test('Go to Definition', async () => {
+            // adding retry to fix https://github.com/eclipse/che/issues/17411
```

Comment:
This issue is closed for some time. I'm not against leaving the code, but I think this line should be removed.

Reply:
@Katka92 yeah, already discussed with Ihor :D When you look at the PR that fixed and closed the issue, the duplication was the solution :D https://github.com/eclipse/che/pull/17397/files#diff-4fe66ae8f83a52cf445794c4dab4b8063ca61a7a32d42fc788f3a05caabbbab2

Reply:
So you want to keep line 80 to remind that this is a fix for some issue, right?

Reply:
yes, pretty much.

Reply:
fixed.